frontend/src/app/application-forms/fields/organization-name.component.spec.ts

```diff
@@ -4,18 +4,19 @@ import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { OrganizationNameComponent } from './organization-name.component';
 import { alphanumericValidator } from '../validators/alphanumeric-validation';
 import { ApplicationFieldsService } from '../_services/application-fields.service';
-import { TestService } from '../../_services/test.service';
 
 describe('Organization Name Component', () => {
   let component: OrganizationNameComponent;
   let fixture: ComponentFixture<OrganizationNameComponent>;
   let formBuilder: FormBuilder;
-  let testService: TestService;
 
   beforeEach(
     async(() => {
-      testService = new TestService();
-      testService.configureTestingModule([OrganizationNameComponent], [FormBuilder, ApplicationFieldsService]);
+      TestBed.configureTestingModule({
```

Comment:
was there are reason for going back to the test service? is it because each of the initializations  require different services?

Reply:
The test service was created before we excluded specs from code climate. It's not super helpful because initializations are different for each spec